bottle-configs/qldbshell.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "qldbshell",
-  "version": "2.0.9",
+  "version": "2.0.0-alpha4",
```

Comment:
is this version number correct? Seems to go from a higher version number to an older one.